Sort/bubbleSort.js

```diff
@@ -0,0 +1,34 @@
+// BubbleSort
+// Time Complexity O(n^2)
+// Space Complexity O(1)
+
+// Run the code to see detailed output
+const array = [64, 25, 12, 22, 11];
+
+function bubbleSort(arr) {
+  let length = arr.length;
+  console.log("Initial array", arr);
```

Comment:
Since you are passing the `arr` here it will out put the sorted array ad 'initial array'. Instead of passing the array directly, please spread the array here (like `[...arr]`

Reply:
10th line is logging Initial Array only. ` [...arr]` is like making a copy of the same.